muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-frontpage.js

```diff
@@ -2,9 +2,97 @@
 
 (function() { 'use strict';
 
+  $.widget("custom.workspaceTeachers", {
+    options: {
+      workspaceEntityId: undefined
+    },
+    _create : function() {
+      this.element.on('click', '.workspace-teacher-info.message', $.proxy(this._sendMessage, this));
+      
+      mApi().user.staffMembers.read({
+        workspaceEntityId: this.options.workspaceEntityId,
+        properties: 'profile-phone,profile-vacation-start,profile-vacation-end'
+      }).callback($.proxy(function (err, staffMembers) {
+        if (!err && staffMembers) {
+          staffMembers.sort(function(a, b) {
+            var an = a.lastName + ' ' + a.firstName;
+            var bn = b.lastName + ' ' + b.firstName;
+            return an < bn ? -1 : an == bn ? 0 : 1;
+          });
+          for (var i = 0; i < staffMembers.length; i++) {
+            var props = staffMembers[i].properties;
+            if (props['profile-vacation-start'] && props['profile-vacation-end']) {
+              var begin = moment(props['profile-vacation-start']).startOf('day');
+              var end = moment(props['profile-vacation-end']).startOf('day');
+              var now = moment().startOf('day');
+              if (now.isSame(begin) || now.isBefore(begin) || now.isBetween(begin, end)) {
```

Comment:
This doesn't show anything on the last day of vacation. `now.isSame(end) || now.isBefore(end)` should do the trick.